server/app/routes/v1/services/service_container_logs.rb

```diff
@@ -5,7 +5,7 @@
     r.is do
       scope = @grid_service.container_logs
 
-      scope = scope.where(name: r['container']) unless r['container'].nil?
```

Comment:
Added `instance_number` field to ContainerLog model. That field is used when querying logs for instance.